plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -119,9 +114,32 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
         return null
     }
 
+    /** Returns the output of the command executed in the container. */
+    fun execWithOutput(params: List<String>): String =
+        commandWithOutput(*getDockerRunArgs(params).toTypedArray())
```

Comment:
I prefer expressions like this to only exist on one line. If it spills onto the next line then it should be a code block.

Reply:
Done.